how-to/what-is-vm-cache-and-how-do-I-use-it.md

```diff
@@ -0,0 +1,31 @@
+# What Is VMCache and How To Enable It
+
+### What is VMCache
+VMCache is a new function that creates VMs as caches before using it.
+It helps speed up new container creation.
+The function consists of a server and some clients communicating
+through Unix socket.  The protocol is GRPC in protocols/cache/cache.proto.
```

Comment:
- Nit: This should really be "gRPC" (throughout the document).
- Could you make `protocols/cache/cache.proto` a link to https://github.com/kata-containers/runtime/blob/master/protocols/cache/cache.proto? That won't be valid until https://github.com/kata-containers/runtime/pull/1166 lands of course but makes it easier for users to find.

Reply:
> * Nit: This should really be "gRPC" (throughout the document).
> * Could you make `protocols/cache/cache.proto` a link to https://github.com/kata-containers/runtime/blob/master/protocols/cache/cache.proto? That won't be valid until [kata-containers/runtime#1166](https://github.com/kata-containers/runtime/pull/1166) lands of course but makes it easier for users to find.

Fixed.